### DISCUSSION TITLE
Reproducing WiX Toolset Security Vulnerability to Verify Upgrade

### DISCUSSION THREAD

**Discussion by Sandhiya-Sivakumar** [wixtoolset/issues#8095] [Questions]
Dear @robmen,

We have recently become aware of a security vulnerability in the **WiX Toolset v3.14.0** that we are currently using. Specifically, the vulnerability involves the handling of temporary files by the Burn component, as described [here](https://github.com/wixtoolset/issues/security/advisories/GHSA-rf39-3f98-xr7r).

So we are in the process of **upgrading to WiX v3.14.1**, In order to ensure that the upgrade procedure we follow addresses the vulnerability, we are seeking assistance in reproducing the issue ourselves. Unfortunately, the ⁠[PoC](https://myvs.download.prss.microsoft.com/dbazure/en_visual_studio_enterprise_2015_with_update_3_x86_x64_dvd_8923288.iso?t=8132cd54-4b83-4478-8b73-fd9eb93437bf&P1=1709239640&P2=601&P3=2&P4=iorgKPv%2bG8n2NANTPUVoB92rr8t3W4XM594%2f9BtQQJrYrr8SwxGDxV%2fj%2f2F6Ulto0bXrIaFoZUr4yV37YAsOZVpM29IMtQEO0673AbDVuTe93qDb6wb7xdlpZSse0LZURUwwIFw5cwHQS2ZtvkunXE0osgXtEBT2IzVbPwVH39%2fum854xb4e2Dp61wgNrMZcOLLluBbeA3KX1sP3mm7WAWXBvlFiQWEnTfR5XH5mlLyPy2qfqCXWCjl84jNX7uY%2bpLR1IbfeD2JlcIQNeW2QrvmmqRrRbGvvaCA97IaSjM16XcDqVjvAEGW3sWXUc7y%2fEf68WZIyT7iilaEDUvaqqA%3d%3d&su=1) link provided in the security update is inaccessible to us.

Could you please provide guidance on how we might reproduce or access the PoC? Our intention is to gain a deeper understanding of the vulnerability and check whether the upgrade procedure resolves the vulnerability before implementing in production.

Any assistance or insights into this matter would be greatly appreciated.

Thanks.

**Comment by robmen** [comment]
It's not just you; the link to the `PoC` was never accessible. However, we do have additional details that we share privately with FireGiant customers. We will not publicly publish additional details about the vulnerability, for obvious reasons.
